tests/build_tests/super_errors/expected/moreArguments4.res.expected

```diff
@@ -10,4 +10,5 @@
   The function has type:
   (int, ~b: int, ~c: 'a, ~d: 'b) => int
   
-  - Missing arguments that must be provided: ~d, ~c, ~b
\ No newline at end of file
+  - Missing arguments that must be provided: ~d, ~c, ~b
+  - Hint: Did you want to partially apply the function? You can do that by putting `...` just before the closing parens of the function call
```

Comment:
Would it be possible to give a code example using the actual function and args the user tried to apply?

```
- Hint: Did you want to partially apply the function? You can do that by putting `...` just before the closing parens of the function call:
  f(v, ...)
```

Reply:
@mediremi turns out we're not in the error reporting place where we have the "rewrite the expr and show" facilities we normally have. But, I added a generic example that I think is good enough: https://github.com/rescript-lang/rescript/pull/7807/commits/c137929a8f359e6e054f2177bd333e845960750f

What do you think?

Reply:
I'm happy with that :+1: